### PR TITLE
build: upgrades build pipeline from CircleCI 1.0 -> 2.0

### PR DESCRIPTION
**Pull request by xlson**
ref https://github.com/grafana/grafana/issues/9633

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = function(config) {
 
     webpack: webpackTestConfig,
     webpackServer: {
-      noInfo: true, // please don't spam the console when running in karma!
+      noInfo: false, // please don't spam the console when running in karma!
```
Comment (bergquist) [inline karma.conf.js]: ? :)

**Review by bergquist — APPROVED** [review]
:100: Sweet! 

Looks great.

File: karma.conf.js (hunk 1 above)
Comment (xlson) [reply karma.conf.js]: @bergquist Not sure if we really want that tbh, I can turn it off again. It's a change from one of Torkels commits.

**Comment by xlson** [comment]
Just noticed that codecov isnt working for the frontend tests, looking into it.

**Comment by bergquist** [comment]
I think we should squash these commits before merging.

**Comment by xlson** [comment]
First proper master build: https://circleci.com/workflow-run/6bd73b7f-b73a-4050-9d33-b6095c512126

**Pull request merged**